gitlab/resource_gitlab_group_membership_test.go

```diff
@@ -109,15 +109,15 @@ func testAccCheckGitlabGroupMembershipDestroy(s *terraform.State) error {
 
 		// GetGroupMember needs int type for userIdString
 		userId, err := strconv.Atoi(userIdString)
-		groupMember, resp, err := conn.GroupMembers.GetGroupMember(groupId, userId)
+		groupMember, _, err := conn.GroupMembers.GetGroupMember(groupId, userId)
 		if err != nil {
-			if groupMember != nil && fmt.Sprintf("%d", groupMember.AccessLevel) == rs.Primary.Attributes["accessLevel"] {
+			if groupMember != nil && fmt.Sprintf("%d", groupMember.AccessLevel) == rs.Primary.Attributes["access_level"] {
```

Comment:
If this is a bug fix, can you add it to your PR description please?

Reply:
sure 👍